webapp/src/services/user.service.js

```diff
@@ -0,0 +1,8 @@
+import http from '../utils/http'
+
+export default {
+    addUser: (user) => http.post(`/user/`, user)
+        .then(response => alert('salvo com sucesso'))
```

Comment:
O serviço não deve ter nenhuma lógica de apresentação. O componente que for utilizar o `user.service.js` deve ser responsável pelo `alert`.

Reply:
Pelo que eu li do código, esse componente ainda não existe. Acho que podemos desconsiderar esse comentário por enquanto.